data/fields/electric_vehicle.json

```diff
@@ -0,0 +1,24 @@
+{
+    "keys": [
+        "bicycle",
+        "moped",
+        "motorcar",
+        "motorcycle",
+        "hgv",
+        "bus",
+        "small_electric_vehicle"
+    ],
+    "type": "manyCombo",
+    "label": "Supported vehicles",
+    "strings": {
+        "options": {
+            "moped": "Mopeds",
+            "bicycle": "E-Bikes",
+            "motorcar": "Cars",
+            "motorcycle": "Motorcycles",
+            "hgv": "Trucks",
+            "bus": "Buses",
+            "small_electric_vehicle": "Small electric vehicles"
```

Comment:
This is a tricky one to word in American English (the default locale): “small electric vehicles” is a legal term in “[many European countries](https://wiki.openstreetmap.org/wiki/Key:small_electric_vehicle)”, but to an American it would sound like a description of, say, a Smart car, because things the size of a bicycle are usually called “devices” rather than “vehicles” here.

Also, this PR translates `hgv` as “trucks” rather than “heavy goods vehicles”, the UK legal term, which I agree with, but it makes the user even less likely to notice that “small electric vehicles” is a legal term.

Assuming it’s OK to use this value even in the U.S., consider either “personal mobility devices” or “micromobility devices”. These terms can also apply to unpowered devices like kick scooters, but I don’t think we need to be concerned about that ambiguity in the context of this field.

Regardless, this string should be in Title Case.

Reply:
Thank you for your feedback. It's sometimes hard for me as a non-native-speaker to find the correct wordings.

I had another look to the [wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcharging_station) and now i'm even more confused than before.
While in the [vehicles-section](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcharging_station#Vehicles) the tagging ```small_electric_vehicles=yes``` is used, the tagging ```scooter=yes``` is used in the examples-section (and in the german wiki page). 
And for trucks, the english wiki says ```hgv=yes``` whilst the german wiki says ```truck=yes```

Should i omit these two from the list as there is a quite low usage anyway or should we try to get a clarification for a unique tagging from the community forum?

Thank you also for the hint to "Title Case" convention.

Reply:
Wow, that is confusing! A discussion in the forum would be a good idea, just to be on the safe side and raise awareness.

Reply:
Don't care about the German Wiki. The quoted information is from the first version of the article from 10 years ago and was no longer up to date. I have just updated the wiki at this point. The term "scooter" in OSM refers to more "moped" like vehicles (which is also how it was written in the translation) rather than "kick scooters" (which are `small_electric_vehicles` if equipped with an electric motor). `truck` should explicitly not be used and replaced with `hgv`.

Reply:
> personal mobility device

I think this term might be easily confused with [_personal mobility aid_](https://agis.com.sg/personal-mobility-aidpma/) devices (i.e. "fancy" electric wheelchairs).

The [wiki](https://wiki.openstreetmap.org/wiki/Key:access) uses "_Electric scooter_" as a description for the `small_electric_vehicle` tag. Maybe that's good enough, especially if we're also thinking about simplifying `hgv` to _Truck_?!